posts/2022-02-14-crates-io-snapshot-branches.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Crates.io Index Snapshot Branches Moving"
+author: the crates.io team <https://www.rust-lang.org/governance/teams/crates-io>
+release: false
+---
+
+Every so often, the [crates.io index](https://github.com/rust-lang/crates.io-index)'s Git history
+is [squashed into one
+commit](https://internals.rust-lang.org/t/cargos-crate-index-upcoming-squash-into-one-commit/8440)
+to minimize the history Cargo needs to download. When the index is squashed, we have saved snapshot
+branches to preserve the history of crate publishes.
+
+Those branches are using server resources because of the recompression of objects that are
```

Comment:
```suggestion
to minimize the history Cargo needs to download. When the index is squashed, we save snapshots
to preserve the history of crate publishes.

Currently, those snapshots are stored as branches in the main index Git repository. This uses server resources because of the recompression of objects that are
```